.changeset/smart-drinks-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+"create-cloudflare": patch
+"wrangler": patch
```

Comment:
how come we're bumping wrangler as well? all changes are in the C3 package

Reply:
Oops, good catch!